Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -45,20 +47,11 @@ export function documentOwnershipByIdSql(documentId: string): string {
  * @returns SQL query string to determine document existence
  */
 export function checkDocumentExistsSql(documentId: string): string {
-  // return format(`SELECT document_id FROM meadowlark.documents WHERE document_id = %L;`, [documentId]);
   return format(`SELECT exists (SELECT 1 FROM meadowlark.documents WHERE document_id = %L LIMIT 1);`, [documentId]);
 }
 
-/**
- * Checks if this document is referenced by other documents
- * @param documentId Document id to check for references
- * @returns SQL query string to determine references existence
- */
-export function checkIsReferencedDocumentSql(documentId: string): string {
-  return format(`SELECT exists (SELECT 1 FROM meadowlark.references WHERE referenced_document_id = %L LIMIT 1);`, [
-    documentId,
-  ]);
-}
+export const checkForReferencesByDocumentId = (documentId: string[]): string =>
+  format(`SELECT existence_id FROM meadowlark.existence WHERE existence_id IN (%L)`, documentId);
```

Comment:
Maybe rename to the former `existenceIdsForDocument`

Reply:
done